node/modules/peering/peering.go

```diff
@@ -0,0 +1,45 @@
+package peering
+
+import (
+	"github.com/ipfs/go-ipfs/peering"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type PeeringPeer struct {
+	ID        string   `json:"ID"`
+	Addrs     []string `json:"Addrs"`
+	Connected bool     `json:"Connected,omitempty"`
+}
+
+type EstuaryPeeringService struct {
+	*peering.PeeringService
+}
+
+type EstuaryPeeringNotifee peering.PeeringService
```

Comment:
can remove, not used anywhere

Reply:
I thought about this. I initially think we should at least prepare this if we want estuary to monitor incoming connection but since we don't have anything concrete plans yet, we can remove.